integration-tests/tests/src/tests/sync/client-reset.ts

```diff
@@ -456,9 +466,10 @@ function getSchema(useFlexibleSync: boolean) {
         });
       });
 
-      it.skip(`handles discard local simulated client reset with ${getPartialTestTitle(
+      it(`handles discard local simulated client reset with ${getPartialTestTitle(
         useFlexibleSync,
       )} sync enabled`, async function (this: RealmContext) {
+        this.timeout(120 * 1000);
```

Comment:
Did it frequently timeout on lower durations?

Reply:
PBS client reset is relative fast so I believe we can remove the line